octodns/provider/constellix.py

```diff
@@ -258,8 +774,10 @@ def _data_for_single(self, _type, records):
         }
 
     _data_for_CNAME = _data_for_single
+    _data_for_PTR = _data_for_single
```

Comment:
`_data_for_PTR`'s data structure is not the same as `_data_for_single`. The previous `_data_for_ALIAS` was correct

```
Traceback (most recent call last):
  File "/Users/yzguy/Projects/octodns/env/bin/octodns-sync", line 11, in <module>
    load_entry_point('octodns==0.9.8', 'console_scripts', 'octodns-sync')()
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/cmds/sync.py", line 39, in main
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/manager.py", line 301, in sync
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/manager.py", line 56, in result
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/manager.py", line 237, in _populate_and_plan
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/base.py", line 49, in plan
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/constellix.py", line 417, in populate
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/record/__init__.py", line 99, in new
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/record/__init__.py", line 360, in validate
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/record/__init__.py", line 710, in validate
AttributeError: 'list' object has no attribute 'endswith'
make: *** [noop] Error 1
```